phonenumber_field/locale/es/LC_MESSAGES/django.po

```diff
@@ -5,40 +5,42 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-05-28 13:04-0700\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"POT-Creation-Date: 2020-09-30 10:42-0500\n"
```

Comment:
Not really sure what the Creation-Date is used for, but I believe it could remain at the old value?

Reply:
This field is auto-generated (not sure if `django` or `poedit`).  
It's merely informative, and not actively used.

Reply:
I believe poedit changed it. Since the po file existed before, why not leave the existing creation date?